doozer/doozerlib/cli/scan_sources_konflux.py

```
@@ -1060,6 +1064,173 @@ async def scan_builders_changes(self, image_meta: ImageMetadata):
+        with tempfile.NamedTemporaryFile(delete=True, suffix='.yaml') as temp_file:
+            try:
+                await artcommonlib.util.download_file_from_github(
+                    repository=rebase_repo_url,
+                    branch=rebase_commitish,
+                    path=art_yaml_path,
+                    github_client=get_github_client_for_org(org),
+                    destination=temp_file.name,
+                )
+                with open(temp_file.name) as f:
+                    return yaml.safe_load(f.read())
+            except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)